apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js

```diff
@@ -436,6 +442,48 @@ describe('DetailViewContents', () => {
       });
     }
 
+    it('appends auto email text if set to true', () => {
+      detailView = mountDetailView('Teacher', {
+        applicationData: {
+          ...DEFAULT_APPLICATION_DATA,
+          status: 'unreviewed',
+          scholarship_status: null,
+          update_emails_sent_by_system: true
+        },
+        viewType: 'teacher'
+      });
+      let options = detailView.find('#DetailViewHeader select').find('option');
+      let applicationStatuses = Object.values(
+        getApplicationStatuses('teacher', true)
+      );
+      var i = 0;
+      options.forEach(option => {
+        expect(option.text()).to.equal(applicationStatuses[i]);
+        i++;
+      });
```

Comment:
Is there a better way to compare values of 2 objects in Javasciprt? (I'm not an expert in Javascript 😅 )

Reply:
The benefit will be you don't have to duplicate this code in here and the test below.

Reply:
Seems like Lodash library has a function for that https://lodash.com/docs#isEqual. (We use Lodash a lot in our client code.)